bib/yoshitom.bib

```diff
@@ -0,0 +1,18 @@
+@inproceedings{panteater2017bandwidth,
```

Comment:
The only BibTeX entries in this file are the exemplary ones that
should not have remained here in the first place. If there are no
publications to be included in this file, keep it empty or delete
it altogether.